Specs/Core/CheckSpec.js

```diff
@@ -190,6 +190,32 @@ defineSuite([
         });
     });
 
+    describe('Check.typeOf.number.equals', function () {
+        it('throws if value associated with mockName1 and/or mockname2 are not a number', function () {
```

Comment:
Small change: no need to mention mockName1 or mockname2. Just say `throws if either value is not a number`.